x-pack/legacy/plugins/file_upload/public/util/file_parser.test.js

```diff
@@ -61,26 +61,4 @@ describe('parse file', () => {
     // Confirm preview function called
     expect(previewFunction.mock.calls.length).toEqual(1);
   });
-
-  it('should use object clone for preview function', () => {
```

Comment:
I would just preserve the test but flip the expectation. We're not expecting a clone, rather an identical reference.

Reply:
I'm not sure it helps us anymore. The preview function is really now just a normal function call with normal args. If I were a developer looking at this code later and I saw a test checking to make sure one arg isn't a clone but rather the same object, I'd wonder "well, why would it be a clone?"

Reply:
Might prevent them from introducing deepclone for some unrelated reason, without checking why this LOC was changed (?) anyway, your call!